site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "Running ZAP on a raspberry pi"
+summary: >
+  Setting up ZAP on the raspberry pi.
+images:
+- https://www.zaproxy.org/blog/2022-08-25-zap-on-raspberry-pi/images/Screenshot_2022-08-20-18-01-55-325_com.realvnc.viewer.android.jpg
+type: post
+tags:
+- blog
+- gsoc
+date: "2022-08-25"
+authors: 
+    - eingengraou
```

Comment:
You also need to update [authors.yml](https://github.com/zaproxy/zaproxy-website/blob/master/site/data/authors.yml) and optionally include a picture of yourself or an image you'd like to use for yourself.